dev-requirements.txt

```diff
@@ -11,4 +11,5 @@ pdfrw==0.4
 watchdog==0.9.0
 pytest===5.3.2
 pytest-django===3.9.0
-factory-boy===2.12.0
\ No newline at end of file
+factory-boy===2.12.0
```

Comment:
can we also bump pylint to 2.6.0 ? pylint 2.5.2  dependes on an older version of isort and makes pip install throw an error